src/trans-word-level/property.cpp

```diff
@@ -87,14 +88,13 @@ void property(
     property_expr.id() == ID_sva_nexttime ||
     property_expr.id() == ID_sva_s_nexttime)
   {
-    prop_handles.resize(no_timeframes, true_exprt());
     if(no_timeframes > 0)
```

Comment:
Not really for this PR, but still wondering: is `no_timeframes == 0` actually a legitimate case at all? Should this perhaps be ruled out via a `PRECONDITION`?

Reply:
Yes, I'll make it a PRECONDITION. BMC without any timeframes is not a thing.